include/vcpkg/base/message-data.inc.h

```diff
@@ -494,6 +494,15 @@ DECLARE_MESSAGE(CmdDependInfoOptSort,
 DECLARE_MESSAGE(CmdEditOptAll, (), "", "Open editor into the port as well as the port-specific buildtree subfolder")
 DECLARE_MESSAGE(CmdEditOptBuildTrees, (), "", "Open editor into the port-specific buildtree subfolder")
 DECLARE_MESSAGE(CmdEnvOptions, (msg::path, msg::env_var), "", "Add installed {path} to {env_var}")
+DECLARE_MESSAGE(CmdExportEmptyPlan, (), "", "Export plan cannot be empty. Install packages before exporting.")
```

Comment:
This needs to be reworded now that it's user facing ("What's a plan?"). Perhaps:

```suggestion
DECLARE_MESSAGE(CmdExportEmptyPlan, (), "", "Refusing to create an export of zero packages. Install packages before exporting.")
```